addon/models/address.js

```diff
@@ -16,21 +37,30 @@ export default class Address extends XMLModel {
         street: String,
         houseNumber: Number,
         swissZipCode: Number,
+        swissZipCodeAddOn: Number,
+        addressLine2: String,
         town: String,
-        country: String,
+        country: Country,
       },
     });
   }
 
   static template = `
   <address>
+    {{#if (not (eq model.country.countryNameShort "ch"))}}
```

Comment:
`(not-eq ...)`

Reply:
Do we have a `not-eq` helper for our templates?

Reply:
Ah true, no we dont.